src/style.css

```diff
@@ -0,0 +1,146 @@
+* {
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+}
+
+.product-card {
+  display: flex;
+  flex-direction: column;
+  width: 200px;
+
+  padding: 32px 16px 0;
+
+  font-family: "Roboto", sans-serif;
+  font-style: normal;
+
+  background: #fff;
+  border: 1px solid #f3f3f5;
+  border-radius: 5px;
+
+  --main-color: #060b35;
+  --secondary-color: #616070;
+  --button-color: #00acdc;
+}
+
+.product-card__image {
+  margin: 0 auto;
+
+  height: 134px;
+  width: 160px;
+}
+
+.product-card__product-name {
+  margin: 40px 0 0;
+
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 18px;
+  color: var(--main-color);
+}
+
+.product-card__product-code {
+  margin: 4px 0 0;
+
+  font-size: 10px;
+  font-weight: normal;
+  line-height: 14px;
+  color: var(--secondary-color);
+}
+
+.product-card__product-rating {
+  margin: 16px 0 0;
+
+  display: flex;
+  justify-content: space-between;
+}
+
+.product-card__reviews-number {
+  margin: 0;
+
+  font-size: 10px;
+  font-weight: 400;
+  line-height: 14px;
+}
+
+.product-card__pricing {
+  margin: 24px 0 0;
+
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: center;
+
+  line-height: 18px;
+}
+
+.product-card__price-word {
+  margin: 0;
+  font-size: 12px;
+  font-weight: 400;
+  color: var(--secondary-color);
+}
+
+.product-card__price-value {
+  margin: 0;
+  font-size: 16px;
+  font-weight: 700;
+}
+
+.product-card__cta-button {
+  margin: 16px 0;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+
+  width: 166px;
+  height: 40px;
+
+  background-color: var(--button-color);
+  border: 1px transparent;
+  border-radius: 5px;
+
+  text-transform: uppercase;
+  text-decoration: none;
+
+  font-size: 14px;
+  font-weight: 700;
+  line-height: 16px;
+  color: #fff;
+}
+
+.product-card__cta-button:hover {
+  background-color: #fff;
+  border: 1px solid var(--button-color);
+  border-radius: 5px;
```

Comment:
we already have border radius, and we dont want to change it on hover, so please remove this property